conmon/cmsg.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2016 SUSE LLC
```

Comment:
Well, you could actually automate it because according to US copyright laws only strings formatted in a certain way are considered to be a legally enforceable copyright statements.

However, to make this simpler do you want to do something like how Kubernetes handles it?

```
Copyright 2015 The Kubernetes Authors.
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
```